workerpool/example/consumer.go

```diff
@@ -28,7 +43,7 @@ EXIT:
 			if !ok {
 				break EXIT // channel closed
 			}
-			c.completed++
+			once.Do(c.rampUp)
```

Comment:
To note, this doesn't need to be only an example of ramp up but also a prestart function that only runs once for each Consumer.

Reply:
Add that as a comment before the `once.Do()`.